lib/client.js

```diff
@@ -158,6 +158,128 @@ class Client {
     const relativePathWithQuery = `${params.resourceType}?${new URLSearchParams(params.searchParams)}`;
     return this.httpClient.get(relativePathWithQuery);
   }
+
+  /**
+   * Create a resource by FHIR id.
+   *
+   * @example
+   * const newPatient = { resourceType: 'Patient',
+   *                      active: true,
+   *                      name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+   *                      gender: 'female',
+   *                      birthDate: '1948-04-14',
+   *                    }
+   *
+   * // Using promises
+   * fhirClient.create({ resourceType: 'Patient',
+   *                     body: newPatient,
+   *                   })
+   *           .then((data) => { console.log(data); });
+   *
+   * // Using async
+   * let response = await fhirClient.create({ resourceType: 'Patient',
+   *                                          body: newPatient,
+   *                                         })
+   * console.log(response);
+   *
+   * @param {Object} params - The request parameters.
+   * @param {string} params.resourceType - The resource type (e.g. "Patient", "Observation").
+   * @param {string} params.body - Information about the patient.
```

Comment:
Should we call this `params.resource`?

Reply:
Also it's not just the patient. I think:

```js
* @params {string} params.resource - The new resource data to create
```

Reply:
Also, since the data has `resourceType` in it, could we just infer it from this data (instead of a separate `resourceType` param)?

Reply:
OK, I'll update the examples and method calls to use `params.resource` when appropriate